project3/styles.css

```
@@ -0,0 +1,218 @@
+:root{
+  --color-red:
+}
+*, 
+*::before,
+*::after{
+  box-sizing: border-box;
+}
+
+body{
+  background-color: #163346;
+}
+
+#Shield {
+    height: 239px;
+    width: 240px;
+    color: #000000;
+}
+  h1{
+  font-family: Teko;
```

Missing fallback font info:
```suggestion
  font-family: Teko, sans-serif;
```